pkg/endpoint/endpoint.go

```diff
@@ -1544,3 +1544,15 @@ func APICanModify(e *Endpoint) error {
 	}
 	return nil
 }
+
+func (e *Endpoint) getIDandLabels() string {
+	e.Mutex.RLock()
+	defer e.Mutex.RUnlock()
+
+	labels := ""
+	if e.SecLabel != nil {
+		labels = e.SecLabel.Labels.String()
+	}
+
+	return fmt.Sprintf("%d (%s)", e.ID, labels)
```

Comment:
Maybe the labels are not important. I would replace it with the security ID + container ID (or perhaps name) to be less verbose.

Reply:
I think the labels are absolutely ideal as they typically contain the application name. Happy to add the container ID but I think it will matter less and less in a cluster enviroment where such IDs are meaningless as they have a scope of a node.

Reply:
On second thought, we should try to hide runtime specific identification such as the container ID as we are very likely to handle non containerised endpoints in the future.